Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -476,6 +476,9 @@ private void TrySetParam(DB.Element elem, BuiltInParameter bip, double value, st
       }
     }
 
+
```

Comment:
Since the only changes to this file is whitespace, could you remove it? Makes for a cleaner PR :)